### ISSUE TITLE
Attach in cucumber not worked

### ISSUE THREAD

**Issue by brunobatista25** [cucumber/cucumber-ruby#1420]
**Attach in cucumber not worked**
hi i'm using cucumber 4.0.0

I configured my hooks for insert a printscreen in my report, but not work, look my example:

```
After do |scenario|
    scenario_name = scenario.name.gsub(/\s+/,'_').tr('/', '_')
    if scenario.failed?
        take_a_picture(scenario_name.downcase!, 'failed')
    else
        take_a_picture(scenario_name.downcase!, 'passed')
    end
end
```

```
module Helper
    def take_a_picture(file_name, result)
        file_path = "report/screenshots/test_#{result}"
        printscreen = "#{file_path}/#{file_name}.png"
        page.save_screenshot(printscreen)
        attach(printscreen, 'image/png')        
    end
end
```
but i open my report print screen not embed the print look the report

![Screenshot from 2020-06-05 11-44-00](https://user-images.githubusercontent.com/8673550/83890262-ec64a680-a721-11ea-9607-761c2e851986.png)

I'm doing something wrong?

**Comment by karamosky** [comment]
Hi @brunobatista25 
Try calling attach with a file handler 
```ruby 
file = File.open(printscreen)
attach(file, 'image/png')
```
Or by passing the file content directly
```ruby
content = File.read(printscreen)
attach(content, 'image/png')
```

**Comment by brunobatista25** [comment]
I try using the two options,but not working
but i open report.html, i see the image is embed in report look the image:

![Screenshot from 2020-06-06 10-25-10](https://user-images.githubusercontent.com/8673550/83945288-17153480-a7e0-11ea-8076-94ca91723b20.png)

but when i open the report in chrome or firefox not show the image look the image report in browser

![Screenshot from 2020-06-06 10-28-49](https://user-images.githubusercontent.com/8673550/83945374-9a368a80-a7e0-11ea-9fab-164032f705f2.png)

**Comment by karamosky** [comment]
Could you try to render the base64 image [here](https://jaredwinick.github.io/base64-image-viewer) and see if the image render correctly?

**Comment by brunobatista25** [comment]
i copy the image that attach generated look this:

![Uploading Screenshot from 2020-06-06 14-15-39.png…]()

the rendering is correctly

**Comment by karamosky** [comment]
🤔 weird! @vincent-psarga any clues on this one?

**Comment by vincent-psarga** [comment]
That's "normal", in the current release, the hooks attachments are not displayed.
We have a fix that has been done but not released yet in `cucumber/react`. This should be fixed in the next release of cucumber.

**Comment by karamosky** [comment]
My bad :) I tought it was already part of this release

**Comment by vincent-psarga** [comment]
Note: normally, there should kot be a new release of cucumber needed, releasing the monorepo should create a new release of `html-formatter-ruby` and a `bundle update cucumber-html-reporter` should be enough.

**Comment by brunobatista25** [comment]
@vincent-psarga @karamosky tanks a lot for helping :).

**Comment by aslakhellesoy** [comment]
@vincent-psarga you mentioned both `html-formatter-ruby` and `cucumber-html-reporter`.

Did you mean `cucumber-html-formatter`? (`https://rubygems.org/gems/cucumber-html-formatter`)

**Comment by vincent-psarga** [comment]
@aslakhellesoy indeed, I fixed that.

**Comment by vincent-psarga** [comment]
This should be fixed with the 4.1.0 release of `cucumber-ruby`

**Comment by emiliamariaa9933** [comment]
could you have the solution for this?
